db/migrate/20161229204445_set_default_content_for_posts.rb

```diff
@@ -0,0 +1,8 @@
+class SetDefaultContentForPosts < ActiveRecord::Migration
+  def up
+    change_column_default :posts, :content, ''
```

Comment:
I'm not too strongly opinionated about *when* it gets merged, so… we could just wait until we plan on doing something else in a migration?